modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/configuration/admin/display/FeatureFlagConfigurationScreen.java

```diff
@@ -48,8 +50,11 @@ public String getCategoryKey() {
 
 	@Override
 	public String getKey() {
-		return FeatureFlagConstants.getKey(
+		String featureFlagKey = FeatureFlagConstants.getKey(
 			_featureFlagType.toString(), getScope());
+
+		return _featureFlagsConfigurationOrder.indexOf(_featureFlagType) +
+			featureFlagKey;
```

Comment:
Declaring the variable `featureFlagKey` it's not necessary, it's single-use. It should be changed to:

```
return _featureFlagsConfigurationOrder.indexOf(_featureFlagType) +
			   FeatureFlagConstants.getKey(
				   _featureFlagType.toString(), getScope());
```

Reply:
Yeah, that's what I thought too, but this one the source formatter made me change to this.

Reply:
Oh well, that's weird. Okay then